Sources/Sentry/Public/SentryScope.h

```diff
@@ -113,6 +121,20 @@ NS_SWIFT_NAME(Scope)
  */
 - (void)addAttachment:(SentryAttachment *)attachment;
 
+/**
+ * Sets the Scope's transaction.
+ *
+ * @param transaction A transaction to add to the Scope.
+ */
+- (void)setTransaction:(id<SentrySpan>)transaction;
```

Comment:
Can we name this `setSpan` instead?

We're setting a span. The reason we have setTransaction in Java still is that we don't want to introduce a breaking change.

Reply:
Python has both https://github.com/getsentry/sentry-python/blob/2df9e1a230f1294b4fc319cb65838dcd6bb2e75c/sentry_sdk/scope.py#L144

`transaction` for active transaction/root span
`span` for active span within the transaction/root span (on Java this is the equivalent of getLatestActiveSpan, we'd need to decide here, otherwise we'd again having the same name for different behaviours

Reply:
This is confusing. In SentrySDK we have startTransaction, in scope it will be called setSpan.

Reply:
Spec says: `setSpan`:

https://develop.sentry.dev/sdk/performance/#scope-changes

So lets do that.